fuel/src/main/kotlin/com/github/kittinunf/fuel/core/FuelManager.kt

```diff
@@ -65,7 +64,7 @@ class FuelManager : RequestFactory, RequestFactory.Convenience {
     private val requestInterceptors: MutableList<FoldableRequestInterceptor> =
             mutableListOf(ParameterEncoder)
     private val responseInterceptors: MutableList<FoldableResponseInterceptor> =
-            mutableListOf(redirectResponseInterceptor(this), validatorResponseInterceptor(200..299))
```

Comment:
I don't feel like using `validatorResponseInterceptor` is a good use of Interceptor anymore. I think I will just take this opportunity and remove it without effect users' behaviors.

Reply:
range from 200 to 299 can be difficult if there is more http status code that considered to be okay.

Reply:
Yeah, agree. After this PR is merged, `!(response.isServerError || response.isClientError)` is the new logic, so it should be okay.